packages/components/src/calendar/style.scss

```diff
@@ -15,8 +15,8 @@
 	position: relative; /* Required to position the navigation toolbar. */
 	box-sizing: border-box;
 	display: inline flow-root;
-	color: colors.$gray-900;
-	background-color: colors.$white;
+	color: $gray-900;
```

Comment:
@mirka can you confirm that this is still the correct way to consume Sass variables? (ie. without `@use` at the top of the file and namespace (ie. `colors.$gray-900` )?

Reply:
`@use` is the correct way. Just, we haven't done the migration work yet 🙃